saga/inMemorySagaLog.go

```diff
@@ -69,3 +69,19 @@ func (log *inMemorySagaLog) GetMessages(sagaId string) ([]sagaMessage, error) {
 		return nil, nil
 	}
 }
+
+/*
+ * Returns all Sagas Started since this InMemory Saga was created
+ */
+func (log *inMemorySagaLog) GetActiveSagas() ([]string, error) {
+	log.mutex.RLock()
```

Comment:
The cost is fairly low, if this isn't a hot spot I'd suggest we follow the idiom.

If this is a hot spot, that sounds like performance optimization. Does it make sense to use the syntax for performance optimizations we discussed?